scripts/ci/publish_wheels.py

```diff
@@ -58,8 +58,8 @@ def publish_notebook_asset() -> None:
                 blob_base = f"version/{wheel_version}"
                 file_base = "extracted/rerun_notebook/static"
                 blobs = [
-                    (bucket.blob(f"{blob_base}/widget.js"), f"{file_base}/widget.js"),
-                    (bucket.blob(f"{blob_base}/re_viewer_bg.wasm"), f"{file_base}/re_viewer_bg.wasm"),
+                    (bucket.blob(f"{blob_base}/notebook/widget.js"), f"{file_base}/widget.js"),
+                    (bucket.blob(f"{blob_base}/notebook/re_viewer_bg.wasm"), f"{file_base}/re_viewer_bg.wasm"),
```

Comment:
When adding this, I forgot that the file will be overwritten by the one generated for the `app.rerun.io` build... which doesn't build with the same flags as the one for the web viewer package and notebook assets.

I think it's a lucky coincidence that this worked on any of the pre-release versions. My solution here is to store the wasm file for the notebook in a separate place.

Reply:
This doesn't break redap, because it hosts its own version of the notebook asset, which is extracted from the wheel. I tried grepping for `widget.js` in other places and didn't find anything else that needed to be changed

Reply:
I would prefer not to do that, just because of the number of other places where we hardcode this 😬...